packages/charts/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -123,13 +124,16 @@ export class Arc extends React.Component<IArcProps, IArcState> {
     return point.callOutAccessibilityData?.ariaLabel || (legend ? `${legend}, ` : '') + `${yValue}.`;
   };
 
+  private _shouldHighlightArc = (activeArc: string[], legend?: string): boolean => {
```

Comment:
>activeArc [](http://example.com/codeflow?start=33&length=9)

this can from this.props. Not needed as argument. All the empty and length checks can move to this. Right?

Reply:
moved